src/index.jsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+import './style.css';
```

Comment:
```suggestion
import React from 'react';
import ReactDOM from 'react-dom';

import App from './App';

import './style.css';
```

처럼 의미를 나누어주면 코드의 가독성이 더 좋지 않을까요?